llvm/test/CodeGen/AMDGPU/GlobalISel/bf16.ll

```diff
@@ -0,0 +1,13790 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
```

Comment:
I think the test should be restricted to argument / return values with bfloats, and check the generated IR translator MIR. Don't copy the full bf16 test, that can just add -global-isel run lines